.github/pull_request_template.md

```diff
@@ -21,5 +21,5 @@
 ### Feature PR Checklist
 - [ ] Documentation (README, local.env.dist, etc.)
 - [ ] Unit tests created or updated
-- [ ] Run `make composershow`
+- [ ] Run `make depsshow`
```

Comment:
I just did a quick search across GitHub and see that `composershow` is used more. Would it make sense to change the Makefile instead?

Reply:
I'm fine with that 👍

Reply:
Okay, done